parlai/scripts/profile_interactive.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Basic script which allows to profile interaction with a model using repeat_query to
+avoid human interaction (so we can time it, only).
+"""
+from parlai.core.params import ParlaiParser
+from parlai.core.agents import create_agent
+from parlai.core.worlds import create_task
+from parlai.agents.repeat_query.repeat_query import RepeatQueryAgent
+
+import random
+import cProfile
+import io
+import pstats
+
+
+def setup_args(parser=None):
+    if parser is None:
+        parser = ParlaiParser(True, True, 'Interactive chat with a model')
+    parser.add_argument('-d', '--display-examples', type='bool', default=True)
+    parser.add_argument('-ne', '--num-examples', type=int, default=5)
+    parser.add_argument(
+        '--display-prettify',
+        type='bool',
+        default=False,
+        help='Set to use a prettytable when displaying '
+        'examples with text candidates',
+    )
+    parser.add_argument(
+        '--display-ignore-fields',
+        type=str,
+        default='label_candidates,text_candidates',
+        help='Do not display these fields',
+    )
+    parser.add_argument(
+        '-it',
+        '--interactive-task',
+        type='bool',
+        default=True,
+        help='Create interactive version of task',
+    )
+    parser.set_defaults(interactive_mode=True, task='interactive')
+    return parser
+
+
+def interactive(opt, print_parser=None):
```

Comment:
can we rename this function to profile_interactive

Reply:
sure